.github/workflows/manual-review.yml

```diff
@@ -185,3 +185,53 @@ jobs:
 
             Review your call to Sentry and see if you can reasonably reduce any
             information that is included, or wait for a VSP review.
+
+  linting:
+    name: Linting
```

Comment:
For better namespacing, labelling, and whatnot, I wonder if this workflow should be called `pull-request.yml`. This particular job is not quite a manual review trigger since it doesn't alert VSP.

```
name: Pull Request
```

Also wondering if this naming would be confusing compared to the "ESLint Check" job. Maybe the other one should be "ESLint Disable Check"?